.github/workflows/test-m1-mac-runner-provisioning.yaml

```diff
@@ -0,0 +1,23 @@
+name: "Provision macOS-arm64 Runner Demo"
+on: release
+jobs:
+  Create_M1_Mac:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./provision-runners
+        with:
+          aws_access_key_id: ${{ secrets.aws_access_key_id }}
+          aws_secret_access_key: ${{ secrets.aws_secret_access_key }}
+          runner_admin_token: ${{ secrets.runner_admin_token }}
+          ami_id: ami-01b8fcd5770ceb9c1
+          ami_user: ec2-user
+          ami_home_directory: /Users
+          instance_type: mac2.metal
+          github_runner_installer: https://github.com/actions/runner/releases/download/v2.298.2/actions-runner-osx-arm64-2.298.2.tar.gz
+          runner_labels: "macOS-arm64"
```

Comment:
nitpick. Maybe naming this parameter `runner_name` will make it easier to see that it's what you should use in the `runs-on`.

I didn't get that until I read the action documentation, and the meaning of the term label may not be clear to people unfamiliar with runners.

Reply:
We could change that, but I'd push back a little bit because that really isn't the runner name. The name if you check the actual available runners is the IP address of the runner. I think the way to handle this is just clear docs.